### DISCUSSION TITLE
Online Trealla playground with code sharing

### DISCUSSION THREAD

**Discussion by guregu** [trealla-prolog/trealla-prolog#337] [Show and tell]
I added a share button to my barebones [Trealla playground](https://php.energy/trealla.html).
Example: https://php.energy/trealla.html?share=fde21b09-81d1-40f0-8808-513f4c43e60c

It's not a very pretty or fancy website, but it may be helpful for sharing Trealla snippets.
In the future I would like to move it to a non-joke website and include a nicer UI.

It's also itself running via Trealla's Wasm port on Fermyon Cloud, code is here: [trealla.html](https://github.com/guregu/php/blob/c2aea9838c7f38eb414d0fa70a7cc63ca826f042/www/public_html/trealla.html#L29-L57), [trealla-share.html](https://github.com/guregu/php/blob/master/www/public_html/trealla_share.html).
Recently I added support for Spin's Postgres/SQLite components, which this uses to save the snippets.

**Comment by infradig** [comment]
Great work, impressed with that you've done here.
